cmd/av/pr_status.go

```diff
@@ -0,0 +1,218 @@
+		variables, err := getQueryVariables()
+		if err != nil {
+			return err
+		}
+
+		client := avgql.NewClient()
+
+		err = client.Query(context.Background(), &query, variables)
+		if err != nil {
+			return err
+		}
+
+		indent := "    "
+		prStatus := query.GithubRepository.PullRequest.Status
+
+		// Print PR info
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			"#",
+			variables["prNumber"],
+			" ",
+			colors.UserInput(query.GithubRepository.PullRequest.Title),
+			"\n",
+		)
+		_, _ = fmt.Fprint(os.Stderr, indent, "Status: ", colors.UserInput(prStatus), "\n")
+
+		if prStatus == "PENDING" || prStatus == "BLOCKED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Status reason: ",
+				colors.UserInput(query.GithubRepository.PullRequest.StatusReason),
+				"\n",
+			)
+		}
+
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Author: ",
+			colors.UserInput(query.GithubRepository.PullRequest.Author.Login),
+			"\n",
+		)
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Created at: ",
+			colors.UserInput(formatTimestamp(string(query.GithubRepository.PullRequest.CreatedAt))),
+			"\n",
+		)
+
+		if prStatus == "QUEUED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Queued at: ",
+				colors.UserInput(
+					formatTimestamp(string(query.GithubRepository.PullRequest.QueuedAt)),
+				),
+				"\n",
+			)
+		}
+		if prStatus == "MERGED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Merged at: ",
+				colors.UserInput(
+					formatTimestamp(string(query.GithubRepository.PullRequest.MergedAt)),
+				),
+				"\n",
+			)
+		}
+
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Base branch: ",
+			colors.UserInput(
+				query.GithubRepository.PullRequest.BaseBranchName,
+				" <- ",
+				query.GithubRepository.PullRequest.HeadBranchName,
+			),
+			"\n\n",
+		)
+
+		// Required checks section
+		_, _ = fmt.Fprint(os.Stderr, "Required Checks\n")
+		requiredCheckStatuses := query.GithubRepository.PullRequest.RequiredCheckStatuses
+		for index := range requiredCheckStatuses {
+			result := requiredCheckStatuses[index].Result
+			requiredCheckName := requiredCheckStatuses[index].RequiredCheck.Pattern
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				emojiForRequiredCheckResult(string(result)),
+				" ",
+				colors.UserInput(requiredCheckName),
+				"\n",
+			)
+		}
+
+		return nil
+	},
+}
+
+func getQueryVariables() (map[string]interface{}, error) {
+	repo, err := getRepo()
+	if err != nil {
+		return nil, err
+	}
+
+	db, err := getDB(repo)
+	if err != nil {
+		return nil, err
+	}
+
+	tx := db.ReadTx()
+
+	currentBranchName, err := repo.CurrentBranchName()
+	if err != nil {
+		return nil, err
+	}
+
+	branch, exists := tx.Branch(currentBranchName)
+	if !exists {
+		return nil, errors.New("could not find current branch")
+	}
+
+	if branch.PullRequest == nil {
+		return nil, errors.New("pull request does not exist")
+	}
+
+	prNumber := branch.PullRequest.Number
+	repository, _ := tx.Repository()
```

Comment:
We actually do need to check if this is set, because if the repo owner/name is empty, this query will fail.